modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -156,17 +143,23 @@ Alternatively, run `rpk topic describe` again to show your reassigned replica se
 ----
 rpk topic describe test -p
 PARTITION  LEADER  EPOCH  REPLICAS  LOG-START-OFFSET  HIGH-WATERMARK
-0          3       2      [1 2 3]   0                 0
-1          2       2      [1 2 3]   0                 0
-2          2       1      [1 2 3]   0                 0
+0          1       2      [1 2 3]   0                 645
+1          1       2      [1 2 3]   0                 682
+2          3       1      [1 2 3]   0                 672
 ----
-
-To cancel an in-progress partition reassignment with the Kafka API, use the flags `--cancel --preserve-throttles`:
-
++
+To cancel all in-progress partition reassignment, use the `move-cancel` command:
++
+[,bash]
+----
+rpk cluster partitions move-cancel
+----
++
+To cancel specific movements to / from a given node, run the following command:
++
 [,bash]
 ----
-kafka-reassign-partitions.sh --bootstrap-server localhost:9092,localhost:9093,localhost:9094,localhost:9095 --reassignment-json-file example.json --cancel --preserve-throttles
-Successfully cancelled partition reassignments for: test-1,test-2
+rpk cluster partitions move-cancel --node 2
 ----
 
 === Differences in partition balancing between Redpanda and Kafka
```

Comment:
@Daisuke, should we now remove this section (_Differences in partition balancing between Redpanda and Kafka_)? If this is is still useful to customers, then please move it up a level, from `===` to `==`. 
Please also move the following section (_Assign partitions at topic creation_) up a level too, from `===` to `==`.

Reply:
Some still look good but some look obsolete. Let me chase.

Reply:
@Deflaimun the preview for this PR still is showing in main, not Beta. Is it OK?

Reply:
@micheleRP , I've updated the differences in https://github.com/redpanda-data/docs/pull/160/commits/16ad33381bc2f7cd217821319be046884d8b3a0f. Who was the original SME for this list? I'd like to have the person involved for review.

Reply:
Nyalia worked on that exact section, and I believe John Spray later contributed. SMEs for other content on this page = @ztlpn and @mmaslankaprv